src/engine/effects/engineeffect.h

```diff
@@ -71,5 +72,4 @@ class EngineEffect final : public EffectsRequestHandler {
     QVector<EngineEffectParameterPointer> m_parameters;
     QMap<QString, EngineEffectParameterPointer> m_parametersById;
 
-    DISALLOW_COPY_AND_ASSIGN(EngineEffect);
```

Comment:
is the point of this PR to make it possible to copy this class? I want to make sure this is ok to remove

Reply:
No, its just already not copy-able because of the `std::unique_ptr` member. So this adheres to the rule of zero. https://en.cppreference.com/w/cpp/language/rule_of_three#Rule_of_zero